docs/users/installation.md

```diff
@@ -60,8 +60,9 @@ RemoveUnused ...
 The first error message means the
 [SemanticDB](https://scalameta.org/docs/semanticdb/guide.html) compiler plugin
 is not enabled for this project. The second error says `RemoveUnused` requires
-the Scala compiler option `-Ywarn-unused`. To fix both problems, add the
-following settings to `build.sbt`
+the Scala compiler option `-Ywarn-unused` (Keep in mind that this compiler
```

Comment:
```suggestion
the Scala compiler option `-Ywarn-unused-import` (or `-Wunused:imports` in 2.13.x)
```

Reply:
Let's provide the exact option to use in 2.13. Most users only care about removing unused imports, not other unused expressions. I sometimes regret having merged the rule to remove unused imports with the rule to remove other unused expressions..

Reply:
I suppose, being explicit is probably better here. Changed this 👍